tasks/elasticsearch-Debian.yml

```diff
@@ -26,3 +26,7 @@
   apt: deb=/tmp/elasticsearch-{{ es_version }}.deb
   when: not es_use_repository
   register: elasticsearch_install_from_package
+
+# ansible uri module requires httplib2
+- name: pip httplib2
+  pip: name=httplib2 extra_args="--user"
```

Comment:
@gingerwizard I have been thoroughly investigating [the above mentioned failure](http://build-eu-00.elastic.co/job/ansible_elasticsearch_acceptance_pr/178/OS=centos-7,es_version=1x,label=metal-pool,suite=config/console) and as always with such things haven't been able to reproduce it locally.

However, the core issue is that the (fedora maintained) EPEL repos are notoriously busy and frequently unavailable. When the role tries to install elasticsearch, in CentOS, yum checks the repo metadata for all defined repos and fails for EPEL. Lots of other people are being affected by it too, e.g. [in this issue](https://github.com/geerlingguy/ansible-role-repo-epel/issues/8) with Jeff Geerling's EPEL galaxy role. Our CI docker images use the EPEL repo as part of their build process as this is required to install ansible among other things.

There are two things we can do:

1. Remove the EPEL and other unnecessary repos for CentOS, referenced by any `yum` operation as a pre-test kitchen step. I will be pushing a PR for that momentarily.

2. As there is no 100% network guarantee for network or repo availability for anything, it would make sense to add a retry mechanism for all `yum install` tasks. See [this example](https://github.com/geerlingguy/ansible-role-repo-epel/issues/8#issuecomment-124218341)
This will shield the role from failing (immediately) when yum faces transient network issues on ec2 and elsewhere.